tutor/src/components/course-calendar/past-assignments.cjsx

```diff
@@ -32,23 +26,18 @@ PastAssignments = React.createClass
     plans = PastTaskPlansStore.get(@props.courseId) or []
 
     <div className='past-assignments'>
-      <EmptyWarning isVisible={isEmpty(plans)} />
-
       {for plan in plans
         <CloneAssignmentLink key={plan.id} plan={plan} />}
     </div>
 
 PastAssignmentsShell = React.createClass
   render: ->
     {courseId} = @props
-
     <LoadableItem
       id={courseId}
       store={PastTaskPlansStore}
       actions={PastTaskPlansActions}
-      load={partial(PastTaskPlansActions.load, {courseId})}
-      renderLoading={ PastAssignmentsLoading }
```

Comment:
hmmm, did the `renderLoading` need to be removed? It was nice having a notice about assignments being loaded to give context for when they do suddenly appear on the page after the request comes back.